com.ibm.streamsx.inet/com.ibm.streamsx.inet.http/HTTPParallelRequest.spl

```diff
@@ -0,0 +1,134 @@
+ * The operator opens on request up to *maxConnections* parallel http connections an sends the results downstream once the 
+ * response is received. The operator has one input stream. The attributes of the input stream may define the kind and the 
+ * parameters of the http operation like method, url, headers  and data. When the response to an request is received, the 
+ * result is passed in an output tuple of the *Result* port. If an http operation fails, the status information and the error
+ * is sent to the *Result* port and optionally to the *Error* port.
+ * The operator maintains a pool of up to *maxConnections* http connections. An incoming request is routed round-robin to 
+ * one of the connections. If the connection is busy the request is queued. The length of the queues is determined with 
+ * parameter *queue*. Finally the congestion policy of the queues is defined with parameter *congestionPolicy*.
+ * 
+
+ * @input	Request				The request Stream with attributes of the http request.
+ * @output	Result				The result stream. One tuple is generated for each input tuple.
+ * 								The minimum type is : tuple<rstring body, list<rstring> headers, rstring message, int32 status, rstring url> where
+ * 								body is the response body, headers is the response header list, message is the error/success message, status is the error status of the operation, url is the encoded url
+ * @output	Error				The optional error stream. One tuple indicates a failed http operation.
+ * 								The type is : <rstring url, rstring message, int32 status>.
+ * 
+ * @param	method				This required parameter determines the kind of the http request. Type HttpMethod
+ * @param	url					The request url. Required parameter.
+ * @param	urlEncode			Defines the function to be used to prepare the url. Useful values are *passThrough* or [function:com.ibm.streamsx.inet.http::urlEncode(rstring)] . Default is urlEncode.
+ * @param	data				The body of the operation. Default is the empty string.
+ * @param	headers				http headers passes in the http request. Default is the empty list.
+ * @param	contentTypeHeader	The content type header. If this header is not empty, it is appended to the headers list. Default is "ContentType: application/x-www-form-urlencoded"
+ * @param	requestTimeout		This is the maximum time in seconds that you allow the transfer operation to take. Normally, name lookups can take a considerable time and limiting operations to less than a few minutes risk aborting perfectly normal operations. Default timeout is 5
+ * @param	connectionTimeout	This is the maximum time in seconds that you allow the connection to the server to take. This only limits the connection phase, once it has connected, this option is of no more use. Default is 20 seconds
+ * @param	width				The number of parallel used threads. This conforms to the maximum number if used http connections. Default is 8
```

Comment:
> This conforms to the maximum number if used http connections.

Doesn't make sense? Not sure what is being said here.

Reply:
Will change the comment to: The number of parallel used threads. The operator opens up to *width* parallel HTTP connections. Default is 8